lib/css/components/_stacks-toggle-switches.less

```diff
@@ -0,0 +1,87 @@
+//
+//  STACK OVERFLOW
+//  TOGGLE SWITCHES
+//
+//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
+//  Stack Overflow. For documentation of all these classes and how to contribute,
+//  visit https://stackoverflow.design/
+//
+//  ============================================================================
+//  $   BASE STYLE
+//  ----------------------------------------------------------------------------
+.s-toggle-switch {
+    display: flex;
+    align-items: stretch;
+
+    //  First hide the radio buttons
+    input[type="radio"] {
```

Comment:
Should we guard these with an explicit class name? Longterm it might be trouble to target the elements in this fashion.

Reply:
My thought here is to minimize the number of things someone has to remember. This is well scoped and should avoid collisions. I'm willing to be wrong here, though.